app.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
+
+var productInfo = [
+  {
+    filepath: 'img/banana.jpg',
+    name: 'banana'
+  },
+  {
+    filepath: 'img/bag.jpg',
+    name: 'bag'
+  },
+  {
+    filepath: 'img/bathroom.jpg',
+    name: 'bathroom'
+  },
+  {
+    filepath: 'img/boots.jpg',
+    name: 'boots'
+  },
+  {
+    filepath: 'img/breakfast.jpg',
+    name: 'breakfast'
+  },
+  {
+    filepath: 'img/bubblegum.jpg',
+    name: 'chair'
+  },
+  {
+    filepath: 'img/chair.jpg',
+    name: 'chair'
+  },
+  {
+    filepath: 'img/cthulhu.jpg',
+    name: 'cthulhu'
+  },
+  {
+    filepath: 'img/dogDuck.jpg',
+    name: 'dogDuck'
+  },
+  {
+    filepath: 'img/dragon.jpg',
+    name: 'dragon'
+  },
+  {
+    filepath: 'img/pen.jpg',
+    name: 'pen'
+  },
+  {
+    filepath: 'img/petSweep.jpg',
+    name: 'petSweep'
+  },
+  {
+    filepath: 'img/scissors.jpg',
+    name: 'scissors'
+  },
+  {
+    filepath: 'img/shark.jpg',
+    name: 'shark'
+  },
+  {
+    filepath: 'img/sweep.png',
+    name: 'sweep'
+  },
+  {
+    filepath: 'img/tauntaun.jpg',
+    name: 'tauntaun'
+  },
+  {
+    filepath: 'img/unicorn.jpg',
+    name: 'unicorn'
+  },
+  {
+    filepath: 'img/usb.gif',
+    name: 'usb'
+  },
+  {
+    filepath: 'img/waterCan.jpg',
+    name: 'waterCan'
+  },
+  {
+    filepath: 'img/wineGlass.jpg',
+    name: 'wineGlass'}
+];
+var allProductChoices = []; //Array for the objects made by constructor
+var randomNumbers = [];
+var lastRandomNumbers = [];
+var counter = 0;
+
+var divEl = document.getElementById('pictures');
+console.log(divEl);
+
+//-------------------------------FUNCTIONS------------------------------------//
```

Comment:
Comments to delineate sections like this is super smart. That way you can find them in the minimap.